packages/app-project/src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -15,49 +15,73 @@ import UnderReviewLabel from './components/UnderReviewLabel'
 const StyledBox = styled(Box)`
   position: relative;
 `
+
 const environment = process.env.APP_ENV
 
-function ProjectHeader ({
+function ProjectHeader({
   availableLocales = [],
   className = '',
   inBeta = false,
   navLinks = [],
-  screenSize = '',
-  title
+  title,
+  width
 }) {
-
   const hasTranslations = environment === 'development' && availableLocales?.length > 1
+  const showDropdownWithRow = (hasTranslations && width < 1200) || (!hasTranslations && width < 1100)
+  const showDropdownWithColumn = (hasTranslations && width < 1000) || (!hasTranslations && width < 900)
+
+  const calcMaxWidth = () => {
+    if (hasTranslations && width >= 1200) {
+      return '560px'
+    } else {
+      return '600px'
+    }
+  }
```

Comment:
[documentation] / [coding style] This section makes perfect sense logically, but I'm mildly concerned that there are a bunch of magic numbers floating around here, which may go unexplained to somebody new who's looking at the code but has never seen this PR.

Recommended actions:
- By default, keep it as is, this isn't a major problem for this PR.
- However, if you can, try to get a second dev opinion to see if they think organising the numbers into something like `const HEADER_WIDTH_FOR_DESKTOP = 1200` is more readable to them.
- Or maybe just add some comments, like `/* Hey we visually tested this and the perfect breakpoint is at width 1200, but be sure to revise this if our general page CSS starts shifting to different breakpoints */`

Reply:
I'll add a note! From the design_thoughts Slack channel, there may be further restyling of ApprovedIcon's placement.